MH4U Dex/CoreDataController.m

```diff
@@ -571,6 +624,7 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator
     }
     
     NSURL *storeURL = [self.applicationDocumentsDirectory URLByAppendingPathComponent:@"MH4U_DEX.sqlite"];
+//    NSLog(@"%@", storeURL);
```

Comment:
Again, I'm going to leave that in for now, because I'll need to uncomment it in the near future.